src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -25,9 +29,12 @@ public FindCommand parse(String args) throws ParseException {
                     String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
         }
 
-        String[] nameKeywords = trimmedArgs.split("\\s+");
+        String[] input = trimmedArgs.split("/",2);
```

Comment:
Will this cause an error if we enter `find /<something>` on the app because `input[0]` and hence `field` will be empty?

Reply:
heads up,Checkstyle error

Reply:
Formatting error for switch case formatting right?

Reply:
No, there should be a spacing between comma and 2.

Reply:
Yes the current implementation  does not support empty field, I will create the relevant exceptions today.

Reply:
> Formatting error for switch case indentation right?

Yes, IntelliJ applied the default indentation. I’m fixing it.